docker/Dockerfile

```diff
@@ -69,6 +73,13 @@ LABEL maintainer="Eddie Antonio Santos <Eddie.Santos@nrc-cnrc.gc.ca>"
 ARG UID_GID=60003
 ARG TINI_VERSION=v0.19.0
 
+RUN set -ex \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends \
+    libfoma0 \
+    git `# for installing python packages direct from github` \
```

Comment:
Is `git` needed in the runtime if the virtualenv is copied over?

Reply:
Nope, good catch.